spec/dynamoid/associations/belongs_to_spec.rb

```diff
@@ -62,4 +62,132 @@
       expect(user.monthly).to eq subscription
     end
   end
+
+  describe "{association}=" do
```

Comment:
Use single quotes for the description strings unless there are escaped characters.